mslib/msui/mscolab_exceptions.py

```diff
@@ -0,0 +1,31 @@
+
+# -*- coding: utf-8 -*-
+"""
+
+    mslib.msui.mscolab_exceptions
+    ~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    Exceptions
+
+    This file is part of MSS.
+
+    :copyright: Copyright 2019- Shivashis Padhi
+    :copyright: Copyright 2019-2024 by the MSS team, see AUTHORS.
+    :license: APACHE-2.0, see LICENSE for details.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
+
+
+class MSColabConnectionError(RuntimeError):
```

Comment:
this is before all imports, when this is important add a comment or someone later refactoring this because of styles it will be moved